<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Asignment10Class</class>
 <widget class="QMainWindow" name="Asignment10Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Asignment10</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <family>MS Serif</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Filter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="FilterListWidget"/>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="font">
               <font>
                <family>Yu Gothic UI Semilight</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Place of origin</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="FilterPlaceLineEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_9">
              <property name="font">
               <font>
                <family>Yu Gothic UI Semilight</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Age</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="FilterAgeLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="filterButton">
            <property name="text">
             <string>Filter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>MS Serif</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Administator</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="AdministratorListWidget"/>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <family>Yu Gothic UI Semilight</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Name</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="NameLineEdit">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <family>Yu Gothic UI Semilight</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Place of origin</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="PlaceLineEdit">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <family>Yu Gothic UI Semilight</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Age</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="AgeLineEdit">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <family>Yu Gothic UI Semilight</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Photograph</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="PhotographLineEdit">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="addButton">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteButton">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="updateButton">
              <property name="text">
               <string>Update</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>MS Serif</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Assistant</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="AssistantListWidge2"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="saveButton">
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="nextButton">
              <property name="text">
               <string>Next</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="ChangeModePushButton">
        <property name="text">
         <string>Change mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="OpenPushButton">
        <property name="text">
         <string>Open external</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>865</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Asignment10.qrc"/>
 </resources>
 <connections/>
</ui>
